.github/workflows/daily-test.yaml

```diff
@@ -0,0 +1,49 @@
+name: Daily build
+
+on:
+  schedule:
```

Comment:
Why daily instead of regular push/PR? There are days (sometimes weeks) passing by when master doesn't change.

Reply:
This was following our discussion in the yt 4 meeting that we didn't want these builds to happen on all PR to reduce the burden from contributors. But I guess we could leave it to push events on master (i.e. when PRs get merged).